### ISSUE TITLE
TTS alt text for page numbers and skip option

### ISSUE THREAD

**Issue by SifAa** [Notalib/flutter_readium#16]
When the tts is reading an ebook and comes to a page number (element with class pagebreak) it should be possible to change what it reads from just the number, to an alt text provided to it like "Here begins page #" as well as to conditionally set it to skip reading the page number altogether

**Comment by ddfreiling** [comment]
`PageBreakBehavior` now has `readAsIs`, `prefixLabel` and `skip` via `reader_tts_preferences.dart`
